site/en/blog/new-in-chrome-87/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: New in Chrome 87
+description: >
+  Chrome 87 is rolling out now! You can now control pan, tilt, and zoom on
+  webcams that support it, range requests and service workers don’t require
+  as many workarounds, the font access API starts it’s origin trial, and
+  plenty more. Let’s dive in and see what’s new for developers in Chrome 87!
+layout: 'layouts/blog-post.njk'
+date: 2020-11-17
+updated: 2020-11-23
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/xFjaDDkGd2mBReqdPxdy.png'
+alt: >
+  New in Chrome hero logo
+---
```

Comment:
@petele could you add:

```
tags:
  - chrome-87
```

Chrome tags are special and you don't need to add them to supportedTags.json.

Reply:
Done.